pygac_fdr/writer.py

```diff
@@ -260,6 +268,7 @@ def __init__(
         Args:
             debug: If True, use constant creation time in output filenames.
         """
+        self.output_dir = output_dir or "."
```

Comment:
Should it be documented somewhere that the default output dir is the current directory?